tests/scripts/manage_titelive_school_related_bookings_and_offers_test.py

```diff
@@ -0,0 +1,78 @@
+import os
+from pathlib import Path
+from unittest.mock import patch
+
+from models import Offer, Booking
+from repository import repository
+from scripts.manage_titelive_school_related_bookings_and_offers import _read_token_from_file, \
+    manage_titelive_school_related_bookings_and_offers
+from tests.conftest import clean_database
+from tests.model_creators.generic_creators import create_user, create_offerer, create_venue, create_stock, \
+    create_booking
+from tests.model_creators.specific_creators import create_product_with_thing_type, create_offer_with_thing_product
+
+
+class ManageTiteliveSchoolRelatedBookingsAndOffersTest:
+    @patch('scripts.manage_titelive_school_related_bookings_and_offers._read_token_from_file')
+    @clean_database
+    def test_should_deactivate_related_offer(self, read_bookings_token_from_file, app):
+        # Given
+        product = create_product_with_thing_type()
+        user = create_user()
+        offerer = create_offerer(siren='775671464')
+        venue = create_venue(offerer, name='Librairie Titelive', siret='77567146400110')
+        offer = create_offer_with_thing_product(venue, product=product, is_active=True)
+        stock = create_stock(offer=offer, price=0)
+        booking = create_booking(user=user, stock=stock, token='AZERTY')
+        repository.save(venue, product, offer, stock, booking, user)
+
+        read_bookings_token_from_file.return_value = [
+            'AZERTY'
+        ]
+
+        # When
+        manage_titelive_school_related_bookings_and_offers('bookings_file')
```

Comment:
```suggestion
        manage_titelive_school_related_bookings_and_offers('fake/path')
```
Ton fake doit comme même démontrer le type de paramètre que tu dois passer.